README.md

```diff
@@ -352,13 +332,29 @@ To release a dataset:
 npm run dataset:release
 ```
 
-
 To weekly release a dataset:
 
 ```
 npm run dataset:scheduler
 ```
 
+## Contributing
+
+Thanks for wanting to contribute! There are different ways to contribute to Open Terms Archive. We describe the most common below. If you want to explore other venues for contributing, please contact us over email (contact@[our domain name]) or [Twitter](https://twitter.com/OpenTerms).
```

Comment:
contact@[our domain name] ?

it it a mistake or a, hidden functionality?

Reply:
I just didn't want to give our email to spambots 😉

Reply:
Do you think it's hard to understand?

Reply:
Well, I did not understand it when I reviewed :-)

So I guess `contact [at] opentermsarchive [dot] org` would be more readable (but I do not know how smart spam bots are ;-))

(And if does not work, it's already in the comments anyway 🤣 )